src/homeScreen.js

```diff
@@ -1,13 +1,26 @@
 const match = require('./match');
+
 const { homeSubScreen } = require('./homeScreen/index');
+const { SettingsScreen } = require('./settingsScreen');
+const { CarbEntryScreen } = require('./carbEntryScreen');
+const { BolusScreen } = require('./bolusScreen');
 
 class HomeScreen {
-    constructor(language) {
-        this.language = language;
+    constructor(language, settingsScreenDefaults) {
+
         this.glucoseScreen = new homeSubScreen.GlucoseScreen(language);
         this.activeInsulinScreen = new homeSubScreen.ActiveInsulinScreen(language);
         this.insulinDeliveryScreen = new homeSubScreen.InsulinDeliveryScreen(language);
         this.activeCarbohydratesScreen = new homeSubScreen.ActiveCarbohydratesScreen(language);
+        this.settingsScreen = new SettingsScreen(language, settingsScreenDefaults);
+        this.bolusScreen = new BolusScreen(language);
+        this.carbEntryScreen = new CarbEntryScreen(language);
+        this.header = new homeSubScreen.Header(language);
```

Comment:
[nit] What is the `homeSubScreen`? It is not clear from the variable name.

Reply:
the sub screens that open off the home screen - like the "Active Insulin" section that opens into its own screen when you click it. I could maybe come up with a better name